cmd/validate/image.go

```diff
@@ -446,7 +443,7 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 				data.output = append(data.output, fmt.Sprintf("%s=%s", applicationsnapshot.JSON, data.outputFile))
 			}
 
-			report, err := applicationsnapshot.NewReport(data.snapshot, components, data.policy, evaluatorData, manyPolicyInput, showSuccesses)
+			report, err := applicationsnapshot.NewReport(data.snapshot, components, data.policy, manyPolicyInput, showSuccesses)
```

Comment:
Since `evaluatorData` is no longer being used, remove it from above as well. This should also cascade in other things being removed. For example, you'll remove this line:

```
evaluatorData = append(evaluatorData, r.data)
```

Look at `r.data`, and remove that too, and whatever populates that, e.g. `res.data = out.Data`. Then keeping doing that until there's nothing else to remove.

Reply:
Similar changes are needed in `cmd/validate/input.go` as well.